dir.targets

```diff
@@ -28,6 +28,15 @@
     <TargetFrameworkProfile></TargetFrameworkProfile>
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
+
```

Comment:
I don't plan to address this as part of this PR.  I'll create an issue tracking following up to determine if this is still needed.